src/scenes/Event/store.js

```diff
@@ -295,10 +305,33 @@ export default class EventStore {
 
   @action
   queryTransactions = async (address) => {
-    this.transactions = await queryAllTransactions(
-      [{ topicAddress: address }],
+    const transactions = await queryAllTransactions(
+      [{ topicAddress: address, status: TransactionStatus.PENDING }],
       { field: 'createdTime', direction: SortBy.DESCENDING },
     );
+    // // this.withdraws = await queryWithdraws(
+    // //   [{ topicAddress: address }],
+    // //   { field: 'block.blockTime', direction: SortBy.DESCENDING },
+    // // );
+
+    // // this.finalizes = await queryResultSets(
+    // //   [{ topicAddress: address, oracleAddress: null }],
+    // //   { field: 'block.blockTime', direction: SortBy.DESCENDING },
+    // // );
+
+    const votes = await queryAllVotes(
+      [{ topicAddress: address }],
+      { field: 'block.blockTime', direction: SortBy.DESCENDING },
```

Comment:
dont need to do any sorting (can remove the orderBy) until all the items are in one array since we want to sort it by a shared field among all the items like `blockTime`